### DISCUSSION TITLE
a question for Mac users about deleting tracks and playlists

### DISCUSSION THREAD

**Discussion by ColmOsiris** [quodlibet/quodlibet#3770] [Q&A] [answered]
apparently, on Windows machines, you can right-click things, and that gives you options like delete. how do i do this on my Mac, please? the equivalent of right-click, ctrl-click, doesn't work for me. neither do alt-click, cmd-click, or shift-click. and i've tried all the combinations.

i have some duplicate tracks i want to get rid of, and also a playlist.

thanks for reading.

**Comment by declension** [comment, marked as answer]
Yes - right clicking or equivalent is the right UI action, so this sounds like a bug on MacOS (related to #3338 again perhaps)
